api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -24,6 +24,9 @@
 
   String WORKSPACE_DELIMITER = "__";
 
+  /** Returns the base path for the FireCloud API. Exposed for integration testing purposes only. */
+  String getBasePath();
```

Comment:
Should you add `@VisibleForTesting` at the interface level too? I don't believe there's any differentiation between unit and other types of testing.

Would it make sense to define this one level up, at some common parent interface for all REST services we depend on? Probably not worth it right now.